app/scripts/lib/crypto/base32.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const base32Decode = require('base32-decode');
+
+function importFxaCryptoDeriver() {
+  return import(/* webpackChunkName: "fxaCryptoDeriver" */ 'fxaCryptoDeriver');
+}
+
+// Base32 encoding based on Douglas Crockford
+// Ref: https://www.crockford.com/wrmg/base32.html
```

Comment:
An article to add https://en.wikipedia.org/wiki/Base32#Crockford.27s_Base32 It felt a bit more useful than that page